docs/tutorials/cloud_fit_deploy/cloud-aws-sagemaker-deployment.md

```diff
@@ -16,7 +16,7 @@ To start using the containers, an inference script and the [wrapper classes](htt
 When authoring an inference [scripts](https://github.com/aws/amazon-sagemaker-examples/blob/master/advanced_functionality/autogluon-tabular-containers/scripts/), 
 please refer to SageMaker [documentation](https://docs.aws.amazon.com/sagemaker/latest/dg/adapt-inference-container.html).
 
-Here is one of the possible inference scripts. 
+Here is one of the possible Tabular inference scripts. 
```

Comment:
Do we need to mention `Tabular` here? Is this tutorial mainly for tabular predictor deployment or general for all predictors? I guess the later one.

Reply:
The tutorial is not only for `Tabular` but this example script is. Basically the idea here is that users can use this example script as a starting point and make some modification to apply on other predictors. I've mentioned it in this way:
```
For inference with other types of AutoGluon Predictors, i.e. TextPredictor, the inference script you provided will be quite similar to the one above.
Mostly, you just need to replace `TabularPredictor` to be `TextPredictor` for example.
```